galaxy.yml

```
@@ -0,0 +1,13 @@
+namespace: "progiuseppe"
+name: "galaxy-repos"
+version: "0.0.3"
+readme: "README.md"
+authors:
+    - "Helena Rasche (https://github.com/erasche)"
```

```suggestion
    - "Helena Rasche (https://github.com/hexylena)"
```

haven't used that username in about 2 years lol

---

I'm truly sorry Helena, copy-pasted it without double checking.

---

Oh haha, no worries @profgiuseppe! no dramas, just had two people ping that handle recently and it's funny to me every time, I don't know where that is persisting that people see it. One of them mentioned it still shows up in gitter somehow, it's just amusing :)